src/js/communities/types.ts

```diff
@@ -0,0 +1,22 @@
+export type CommunityGroup = {
+  readonly name: string;
```

Comment:
nit: can we apply lexicographical sorting on attributes of types; simply a consistency, should not be considered a requirement for approval however

Reply:
Resolved in [553ddb6](https://github.com/newwwie/newwwie.com/pull/117/commits/553ddb6c839d5619d71c513eba742e433405e367)